src/NServiceBus.Core/Unicast/Config/UnicastBus.cs

```diff
@@ -94,9 +92,9 @@ protected internal override void Setup(FeatureConfigurationContext context)
             }
         }
 
-        Unicast.UnicastBus CreateBus(IBuilder builder)
+        static UnicastBusInternal CreateBus(IBuilder builder)
```

Comment:
Can we now remove this along with the registrations above? If we just `ConfigureComponent<UnicastBusInternal>()` it should just work.